package.json

```diff
@@ -1052,7 +1052,7 @@
         "ms-rest-azure": "^2.3.1",
         "pg": "^8.3.3",
         "pg-connection-string": "^2.3.0",
-        "pg-structure": "^7.2.3",
+        "pg-structure": "^6.2.0",
```

Comment:
Can you add more description about why we need an older version? For example, is there a bug we can track that will let us know when we can move to the latest version?

Reply:
pg-structure packages version >6 only support Postgres versions >=10. Created an issue for this #1702